modules/ROOT/pages/dw-core-namespaces.adoc

```diff
@@ -1,11 +1,10 @@
 = Core Namespaces (dw::Core)
 
-
 |===
 | Namespace | Definition | Description
 
 | xsi
-| `xsi =` `+http://www.w3.org/2001/XMLSchema-instance+`
+| `+xsi = http://www.w3.org/2001/XMLSchema-instance+`
```

Comment:
TODO: need DW Jira for manual fixes "+"s in the backticks to avoid link in output